src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -45,7 +45,6 @@ private NewGameChooser(final Frame owner) {
     setupListeners();
     setWidgetActivation();
     updateInfoPanel();
-    refreshGameList();
```

Comment:
Note that on line 53, in the create components method, we have:
`     gameListModel = getNewGameChooserModel();`
and in refreshGAmeList, we have:
` gameListModel = getNewGameChooserModel();`

These are both blocking calls that load all available maps, doubling the map load time.